src/hooks/verify.ts

```diff
@@ -45,7 +49,7 @@ export default class VerifyService implements Hook {
       throw new ReferenceError(`Verification role not found in guild ${message.guild.name}!`);
     }
     message.member
-      .addRole(role)
+      .delRole(role)
```

Comment:
There is no point on using `delRole` because `message.member` will lack the role at that point anyway.